### PR TITLE
De33386/remove search param get ie11

### PR DESCRIPTION
**Pull request by mtjandra**
* Allows loading QE on load, but still cannot apply filters yet

### CODE REVIEW

File: components/d2l-quick-eval/compatability/universal-methods.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const StringEndsWith = (str, search) => {
```
Comment (AlexBedley) [inline components/d2l-quick-eval/compatability/universal-methods.js]: I would just call out that this file only exists because of ie11 and comment what built-in function you are trying to mimic so when we remove ie11 support we can easily revert this change.

File: components/d2l-quick-eval/compatability/universal-methods.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+		return false;
+	}
+
+	return str.substring(str.length - search.length, str.length) === search;
+};
+
+const GetQueryStringParams = (queryString) => {
+	const query = {};
+	if (queryString) {
+		const pairs = (queryString[0] === '?' ? queryString.substr(1) : queryString).split('&');
```
Comment (AlexBedley) [inline components/d2l-quick-eval/compatability/universal-methods.js]: nit: I feel like this deserves 2 lines

File: components/d2l-quick-eval/compatability/universal-methods.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+		const validPairs = pairs.filter(p => p.indexOf('=') >= 0);
+
+		for (let i = 0; i < validPairs.length; i++) {
```
Comment (AlexBedley) [inline components/d2l-quick-eval/compatability/universal-methods.js]: nit: foreach

File: components/d2l-quick-eval/compatability/universal-methods.js (hunk 1 above)
Comment (mtjandra) [reply components/d2l-quick-eval/compatability/universal-methods.js]: I can add a comment to the top of the file in the earlier PR https://github.com/BrightspaceHypermediaComponents/activities/pull/160

This is still WIP

Comment (AlexBedley) [reply components/d2l-quick-eval/compatability/universal-methods.js]: woops got my PRs mixed up. I was wondering why you kept force-pushing haha!
